scripts/table_updates/utilities.py

```diff
@@ -126,12 +188,36 @@ def revert_table_version(syn, table_id):
     """
     table_schema = syn.get(table_id)
     table_columns = syn.getColumns(table_schema.columnIds)
-    table_columns = [col['name'] for col in list(table_columns)]
-    previous_version_num = table_schema.versionNumber-1
-    old_data = syn.tableQuery('SELECT * FROM %s.%s' % (table_id, previous_version_num)).asDataFrame()
+    table_columns = [col["name"] for col in list(table_columns)]
+    previous_version_num = table_schema.versionNumber - 1
+    old_data = syn.tableQuery(
+        "SELECT * FROM %s.%s" % (table_id, previous_version_num)
+    ).asDataFrame()
     old_data = old_data.reset_index(drop=True)
-    table_columns = list(set(table_columns) & set(old_data.columns)) 
+    table_columns = list(set(table_columns) & set(old_data.columns))
     temp_data = old_data[table_columns]
     table_query = syn.tableQuery("SELECT * from %s" % table_id)
     syn.delete(table_query.asRowSet())
-    syn.store(Table(table_schema, temp_data))
\ No newline at end of file
+    syn.store(Table(table_schema, temp_data))
+
+
+def remove_backslash(df: pandas.DataFrame) -> pandas.DataFrame:
```

Comment:
Can we make this column specific. Like have an input parameter of the columns we want to remove backslash for? Because for this table, it's just limited to `drugs_drug_1` and `drugs_drug_2` that have the backslashes.

Reply:
I originally made it column specific and allow column name as an input parameter. But I found it's not very generic since we don't know if other `drugs_drug` columns have `\` in other cohorts data. Based on the data dictionary, 'drugs_drug_1', 'drugs_drug_2', 'drugs_drug_3', 'drugs_drug_4', and 'drugs_drug_5' in ca_directed_drugs could contain backslashes. `drugs_drug_1 and drugs_drug_2` is merely a case for PANC and I'm not sure the columns for cohorts I didn't test for. So I let the function to generate the column list to be updated.

Reply:
Yeah my reasoning is I wonder if we will run into the opposite problem where we could be removing important `\` in datasets. Though not sure how likely it is.

Reply:
@rxu17 do you prefer a format like this [function](https://github.com/Sage-Bionetworks/genie-bpc-pipeline/blob/b329dac146e62497473d0aaeb1c40564650d32ed/scripts/table_updates/utilities.py#L23C1-L37C1) or add a col param to this function?

Reply:
Based on our discussion, we prefer the latter option.